WordPress/src/main/java/org/wordpress/android/ui/ActivityLauncher.java

```diff
@@ -38,16 +49,21 @@ public static void viewCurrentSite(Context context) {
         context.startActivity(intent);
     }
 
-    public static void viewBlogStats(Context context, Blog blog) {
-        if (blog == null) return;
-
+    public static void viewBlogStats(Context context, int blogLocalTableId) {
         Intent intent = new Intent(context, StatsActivity.class);
-        intent.putExtra(StatsActivity.ARG_LOCAL_TABLE_BLOG_ID, blog.getLocalTableBlogId());
+        intent.putExtra(StatsActivity.ARG_LOCAL_TABLE_BLOG_ID, blogLocalTableId);
         context.startActivity(intent);
     }
 
     public static void viewCurrentBlogPosts(Context context) {
+        ActivityLauncher.viewCurrentBlogPosts(context, true);
+    }
+
+    public static void viewCurrentBlogPosts(Context context, boolean animated) {
```

Comment:
We can get rid of the `animated` parameter - that was only necessary when there was a drawer.